countmatrixforedger/countmatrixforedger.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="countmatrixforedger" name="countMatrixForEdger" version="1.0">
+    <description>Make a count matrix for edgeR</description>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command>
+        <![CDATA[ 
+            #for $i, $s in enumerate( $sample )#
+                echo -e ${s.sample_data}"\t"${s.sample_name}"\t"${s.sample_group} >> script.data;
+            #end for#
+
+            $__tool_directory__/tools/scrsh.sh script.data;
+
+            cat count.matrix > $count_matrix;
+            cat annotation > $annotation;
+            cat script.log > $log;
+            cat error.log > $error_log
+
+        ]]>
+    </command>
+    <inputs>
+        <repeat name="sample" title="">
+            <param name="sample_data" type="data" format="data" label="Output matrix from EDGE-pro (out_rpkm)" />
+            <param name="sample_name" type="text" label="" help="Sample name" />
+            <param name="sample_group" type="text" label="" help="Sample group name- it has to be the same string for each sample of this group" />
+        </repeat>
+    </inputs>
+    <outputs>
+        <data name="count_matrix" format="tabular" label="countMatrixForEdger: count matrix"/>
```

Comment:
Add attribute `from_work_dir="count.matrix"` and similarly for the other outputs.
This will save the file copying.